pages/docs/docs.md

```diff
@@ -1222,10 +1222,6 @@ That's because you often do see code like this:
 In LightScript, accessing an index or property using `[]` requires an indent:
 
     node
-      .children
-      [0]
-
-    node
     .children
     [0] // oops!
```

Comment:
I'm guessing some non-ascii invisible whitespace snuck in there somehow? 

Regardless, the `<!-- -->` is a separator used for markdown, not something intended to be part of the lightscript. If you look for it in other parts of the file you'll see how it's used